examples/printerdemo/ui/printerdemo.slint

```diff
@@ -44,8 +44,8 @@ export component MainWindow inherits Window {
 
     callback quit();
 
-    width: 772px;
-    height: 504px;
+    preferred-width: 772px;
+    preferred-height: 504px;
```

Comment:
The problem is that it is not meant to be resized. It will look wrong if resized on the desktop. Is that Ok?